### ISSUE TITLE
Provide clickable links in default verification template

### ISSUE THREAD

**Issue by iparr** [putyourlightson/craft-campaign#303]
The verification link in the default template is not provided as a clickable link.

In many email clients, they will transform this to a link for you. But for some (not sure which ones), they will not.

The following code is available in the raw email:

```
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable

Thank you for subscribing to the mailing list. Please verify your email address by clicking on the following link:
https://domain.com/actions/campaign/forms/verify-subscribe?pid=XXX

Content-Type: text/html; charset=utf-8
Content-Transfer-Encoding: quoted-printable

Thank you for subscribing to the mailing list. Please verify your email address by clicking on the following link:
https://domain.com/actions/campaign/forms/verify-subscribe?pid=XXX
```

Would it be possible for the `text/html` portion to instead read (without having to create a custom template):

```
Thank you for subscribing to the mailing list. Please verify your email address by clicking on the following link:
<a href="https://domain.com/actions/campaign/forms/verify-subscribe?pid=XXX">https://domain.com/actions/campaign/forms/verify-subscribe?pid=XXX</a>
```

This would guarantee a clickable link in all clients (I think!)

I think this could be addressed here, but I'm not sure how to separate the HTML/plaintext versions: https://github.com/putyourlightson/craft-campaign/blob/develop/src/services/FormsService.php#L91

**Comment by bencroker** [comment]
I've yet to come across an email client that does _not_ transform URLs into clickable links. Can you provide any specific examples?

**Comment by bencroker** [comment]
Never mind, I just added in https://github.com/putyourlightson/craft-campaign/commit/bb8ae1ecbfc6f02914de906e9447a6e337d687e4 for the 2.0.0 release.

**Comment by bencroker** [comment]
This has been released in version 2.0.0.
